### PR TITLE
Add contacts to events

### LINKED ISSUE #107: Add Person to events
As an event planner, i want to be able to add persons to events

NOT CONCRETE:
There could probably be a few variations of this command to accomodate cases like:
- just adding a role
- adding multiple roles

Or maybe we only need 1 variation to add 1 role at a time probably works too

### PR DESCRIPTION
**Pull request by cshao02**
closes #107 

Implemented adding of contacts to events with `event-add ei/EVENT INDEX [a/ | s/ | ve/ | vo/]CONTACT INDEX` where event index is compulsory and one of a/ or s/ or ve/ or vo/ prefixes followed by at least one contact index is compulsory. 
e.g. `event-add ei/1 a/1,2 ve/3`

Duplicate indices for the same role is not allowed e.g. `event-add ei/1 a/1,1`, but is allowed for different roles e.g. `event-add ei/1 a/1 ve/1` provided that the contact added has the role that he or she is being added to.

TODO:
update UI to be able to dynamically render the changes to number of attendees etc on adding of contacts.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -191,5 +191,38 @@ public static Event parseEvent(String event) throws ParseException {
...
+
+        for (String str : strings) {
+            Index index;
+            try {
+                index = ParserUtil.parseIndex(str.trim());
+            } catch (ParseException e) {
+                throw new ParseException("One or more of the contact indices is not a non-zero unsigned integer.");
+            }
+
+            if (indices.contains(index)) {
```
Comment (jan-kai1) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:222]: Looks good, allows for addition of multiple indices while preventing duplicates

File: src/test/java/seedu/address/logic/commands/event/commands/AddPersonToEventCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.EventManager;
+import seedu.address.testutil.AddressBookBuilder;
+import seedu.address.testutil.EventBuilder;
+
+public class AddPersonToEventCommandTest {
```
Comment (jan-kai1) [inline src/test/java/seedu/address/logic/commands/event/commands/AddPersonToEventCommandTest.java:33]: Looks good, maybe could add test cases for negative values?

**Review by jan-kai1 — COMMENTED** [review]
LGTM, maybe could test issues related to invalid negative indices?

**Review by shardhrv — APPROVED** [review]
LGTM! Solid work as always!
